packages/itwinui-css/scripts/createComponent.js

```diff
@@ -3,11 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 const fs = require('fs');
-const {
-  copyrightBannerScss,
-  copyrightBannerHtml,
-  copyrightBannerJs,
-} = require('./copyrightLinter');
+const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('./copyrightLinter');
 
 const componentName = process.argv.slice(2).join('-');
 
```

Comment:
Does it still work?

Reply:
I didn't change anything, so it should

Reply:
whoops, I thought you were talking about the copyrightLinter, not createComponent.

this script is fixed now